packages/odyssey-react/src/components/VisuallyHidden/index.tsx

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { FunctionComponent, ReactText } from 'react';
+import { useOmit } from '../../utils';
+import styles from './VisuallyHidden.module.scss';
+
+export type Props = {
+  /**
+   * Text content to be read only to screen readers
+   */
+  children: ReactText,
+};
+
+const VisuallyHidden: FunctionComponent<Props> = (props) => {
+  const {
+    children,
+    ...rest
+  } = props;
+
+  const omitProps = useOmit(rest);
+
+  return (
+    <span
```

Comment:
minor but we may want to revisit this component in the future and allow for different tags to be rendered here via an `as="strong"` or similar API. That way specific inflection can be applied in the SR text that is announced

Reply:
+1 to this, let's log an issue for this feature if we don't plan on getting to it now.

Reply:
Since these aren't in use today, I've created a fast-follow ticket for next sprint: https://oktainc.atlassian.net/browse/OKTA-425816